services/core-api/app/api/now_submissions/models/application.py

```diff
@@ -225,7 +227,11 @@ def __repr__(self):
     @classmethod
     def find_by_now_application_guid(cls, now_application_guid):
         cls.validate_guid(now_application_guid)
-        now_identity = NOWApplicationIdentity.query.filter_by(now_application_guid=now_application_guid).first()
+
+        from app.api.now_applications.models.now_application_identity import NOWApplicationIdentity
```

Comment:
Imports should be at the top of the file

Reply:
it causes a circular import at the top of the file, this works.... i can do some more digging if needed.

Reply:
I guess that's why its a should instead of shall, this seems like a valid reason.